### DISCUSSION TITLE
后空翻

### DISCUSSION THREAD

**Discussion by MiniStrikers** [MiRoboticsLab/cyberdog_ros2#149] [General]
有没有人的铁蛋可以做个后空翻？密码是什么？

**Comment by cattei** [comment]
我试了topic，其他动作可以做，但后空翻不行，感觉被什么锁定了，还在查资料中。小米太坑了，项目完全不更新，文档也没说明白。
我自己记录了些内容，有兴趣可以看下：https://ttip11ywee.feishu.cn/wiki/PFxQwRSdJiVmQNk8l9dcJg9vnIc?from=from_copylink
